lib/kube/proxy/resource_rbac_test.go

```diff
@@ -128,6 +129,31 @@ func TestListPodRBAC(t *testing.T) {
 		},
 	)
 
+	userWithTraits, _ := testCtx.CreateUserWithTraitsAndRole(
+		testCtx.Context,
+		t,
+		usernameWithTraits,
+		map[string][]string{
+			"namespaces": {metav1.NamespaceDefault},
+		},
+		RoleSpec{
```

Comment:
nit: not strictly for this PR but `RoleSpec` is reasonably long, might use a function that returns RoleSpec to reduce loc in this test.